tests/integration_test/action_service_test.py

```diff
@@ -5809,3 +5809,287 @@ def __mock_cache_result(*args, **kwargs):
             [{'text': generated_text, 'buttons': [], 'elements': [], 'custom': {}, 'template': None,
               'response': None, 'image': None, 'attachment': None}
              ])
+
+    @mock.patch.object(GPT3FAQEmbedding, "_GPT3FAQEmbedding__get_answer", autospec=True)
+    @mock.patch.object(GPT3FAQEmbedding, "_GPT3FAQEmbedding__get_embedding", autospec=True)
+    @patch("kairon.shared.llm.gpt3.Utility.execute_http_request", autospec=True)
+    def test_kairon_faq_response_action_with_action_prompt(self, mock_search, mock_embedding, mock_completion):
```

Comment:
this test case does not assert whether response from http action is really being added to prompt!!